packages/core/test/types.test.ts

```diff
@@ -1,5 +1,4 @@
-import Bugsnag, { Client, Config } from '../..'
-import "jasmine"
+import Bugsnag, { Client, Config } from '..'
```

Comment:
These are moved for now. We may wish to keep them to test against the public manually maintained type definitions whilst separate internal types are used for testing internals.